packages/i18n/I18nDecorator/I18nDecorator.js

```diff
@@ -143,12 +143,70 @@ const I18nDecorator = hoc(defaultConfig, (config, Wrapped) => {
 	return I18nDecorator;
 });
 
+/**
+ * Default config for `I18nContextDecorator`.
+ *
+ * @memberof i18n/I18nDecorator.I18nContextDecorator
+ * @hocconfig
+ */
 const contextDefaultConfig = {
+	/**
+	 * The prop name for `locale` property of context.
+	 *
+	 * @type {String}
+	 * @default null
+	 * @public
+	 * @memberof i18n/I18nDecorator.I18nContextDecorator.contextDefaultConfig
+	 */
 	localeProp: null,
+
+	/**
+	 * The prop name for `rtl` property of context.
+	 *
+	 * @type {String}
+	 * @default null
+	 * @public
+	 * @memberof i18n/I18nDecorator.I18nContextDecorator.contextDefaultConfig
+	 */
 	rtlProp: null,
+
+	/**
+	 * The prop name for `updateLocale` property of context.
+	 *
+	 * @type {String}
+	 * @default null
+	 * @public
+	 * @memberof i18n/I18nDecorator.I18nContextDecorator.contextDefaultConfig
+	 */
 	updateLocaleProp: null
 };
 
+/**
+ * A higher-order component that is used to access the properties of the context via props whose names are set as config values.
```

Comment:
I think it would be better if we described it in detail.
```suggestion
 * A higher-order component that is used to access the properties of the i18nContext via props by specifying the `localeProp`, `rtlProp` and `updateLocaleProp` configuration options respectively.
```